osbs/api.py

```diff
@@ -305,7 +302,8 @@ def _create_scratch_build(self, build_request):
             builder_img['kind'] = 'DockerImage'
             builder_img['name'] = ref
 
-        build_config_name = 'scratch-%s' % datetime.datetime.now().strftime('%Y%m%d%H%M%S')
+        build_config_name = 'scratch-%s' % (
+            build_json['spec']['output']['to']['name'][-20:])
```

Comment:
20 here is a magic number - lets not use those or at least use a constant for it.

Why is it even set to 20? Is this the registry hostname part?